opensearch_py_ml/ml_models/sentencetransformermodel.py

```diff
@@ -802,10 +820,16 @@ def save_as_pt(
         :param add_apache_license:
             Optional, whether to add a Apache-2.0 license file to model zip file
         :type add_apache_license: string
+        :param third_party_copyrights_statements: Statements text for non Apache-2.0 licensed third party model. Should be put in the final artifact.
+        :type third_party_copyrights_statements: string
         :return: model zip file path. The file path where the zip file is being saved
         :rtype: string
         """
 
+        if add_apache_license == True and third_party_copyrights_statements is not None:
+            assert (
+                False
+            ), "When the model is from third party under non Apache-2.0 license, we can not add Apache-2.0 license for it."
```

Comment:
My understanding is, we will still distribute the artifact under apache 2.0 license but we also need to add extra attribution file to deliberately mention about the contributors name. Am I missing anything?